packages/sources/src/addSourceWizard/hardcodedComponents/openshift/costManagement.js

```diff
@@ -0,0 +1,222 @@
+import { TextContent, Text, TextVariants, TextListItem, TextList, Button, ClipboardCopy, Popover, ButtonVariant } from '@patternfly/react-core';
+import { global_spacer_md, global_spacer_xl } from '@patternfly/react-tokens';
+import { QuestionCircleIcon } from '@patternfly/react-icons';
+import React from 'react';
+
+export const PrerequisiteDescription = () => (
+    <TextContent key='description'>
+        <Text component={TextVariants.p}>
+            Before you begin, install the following prerequisites if not present:
+        </Text>
+    </TextContent>
+);
+
+export const PrerequisiteOCPText = () => (
+    <TextContent key='prerequisite-ocp'>
+        <Text component={TextVariants.p}>
+            On your OpenShift cluster, install:
+        </Text>
+    </TextContent>
+);
+
+export const PrerequisiteSystemText = () => (
+    <TextContent key='prerequisite-system'>
+        <Text component={TextVariants.p}>
+            On a system with network access to your OpenShift cluster, install:
+        </Text>
+    </TextContent>
+);
+
+export const PrerequisiteOCPList = () => (
+    <TextContent>
+        <TextList component='ul' style={{ marginLeft: global_spacer_md.value, marginTop: `-${global_spacer_md.value}` }}>
```

Comment:
Could we extract this into a classname and define it in css? It would be better to have all styles together.

Reply:
You want me to create `costManagement.styles.js` and use react-styles's css for that?

Reply:
Nope, here https://github.com/RedHatInsights/frontend-components/tree/master/packages/sources/src/styles you can find styles, so add a new css file, add your styles there and then import this file in index.js https://github.com/RedHatInsights/frontend-components/blob/201dfcd4818179a4fe887d698ea92d4e089be5fd/packages/sources/src/index.js#L13, so it will be exported together with the wizard.

Reply:
Thanks that was helpful